api/woeip/settings.py

```diff
@@ -30,7 +30,7 @@
     "django.contrib.gis",
 ]
 
-THIRD_PARTY_APPS = ["django_extensions", "rest_framework", "storages", "drf_yasg"]
+THIRD_PARTY_APPS = ["django_extensions", "rest_framework", "storages", "drf_yasg", "corsheaders"]
```

Comment:
Was this install prompted by a specific error? My understanding is that the [nginx server]( https://github.com/openoakland/woeip/blob/master/nginx-proxy/cors.conf) configures the cors headers. This should be how the maps component is able to make a [get request](https://github.com/openoakland/woeip/blob/994710f174c3412bcb66089452b6ea60bc5cdd87/web/src/components/Map/index.tsx#L49) to the pollutant values without Django CORS Header installed/configured.

Reply:
ooh good call! 

so it was throwing this error before I added the cors config:
![image](https://user-images.githubusercontent.com/18567829/80041632-c6f14580-84b1-11ea-8c45-c59259f413cd.png)

That was also back before I actually got the request to go through, so I had thought that was part of the issue, but it looks like it was actually just a red herring. Didn't realize that the nginx server was taking care of cors stuff for us so we might just need to tweak some config there to handle this error message (even though it doesn't actually cause the request to fail). I'll delete all the cors changes from django, thanks for catching this!